src/main/java/com/kodilla/ecommercee/OrderDto.java

```diff
@@ -0,0 +1,13 @@
+package com.kodilla.ecommercee;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+public class OrderDto {
```

Comment:
Wiem, że tak jest w kursie i wiem, że wielu programistów tak implementuje klasy DTO. Nie kwestionuję tego bo jest to w pełni zgodne z najlepszymi praktykami obiektowo orientowanego programowania. Chciałbym jednak zwrócić tutaj Twoją uwagę na nadmiarowość jaka jest z tym związana. Robert C. Martin (znany jako Uncle Bob) również zwraca na to uwagę w książce "Czysty kod". Sugeruje on nawet, że takie podejście jest stosowane przez purystów a on uznaje to za zbędne. W jego ocenie klasy DTO powinny zawierać jedynie pola, które są publiczne i w tym sensie być zgodne ze starymi strukturami znanymi z takich języków jak C. Enkapsulacja, konstruktory, gettery i settery tutaj w pewnym sensie faktycznie nie dają żadnej wartości poza tym iż są zgodne z podejściem obiektowym. 

Nie zmienia to jednak faktu, że sam tak robię w pracy bo robią tak wszyscy koledzy a czysty kod to również taki, który w całym projekcie jest spójny. Pisząc ten komentarz chcę jedynie otworzyć Twoją głowę na możliwość krytyki tego co ktoś kiedyś powiedział, żebyś robił bo "tak trzeba".

Reply:
Z powodu problemów z lombokiem w CircleCi zmieniłem na "ręczną" implementację metod